matchbox_server/src/main.rs

```diff
@@ -32,39 +26,51 @@ async fn main() {
                 .with_target(false),
         )
         .init();
+}
+
+#[tokio::main]
+async fn main() {
+    // Initialize logger
```

Comment:
I know you're staying in line with the previous code but these comments feel like a bit much

Reply:
You're right, this is self-documenting code.